Common/src/main/java/com/blamejared/crafttweaker/api/command/CommandUtilities.java

```diff
@@ -14,11 +14,14 @@
 import net.minecraft.network.chat.Style;
 import net.minecraft.server.level.ServerPlayer;
 import net.minecraft.world.entity.player.Player;
+import org.apache.logging.log4j.Logger;
 
 import java.nio.file.Path;
 
 public final class CommandUtilities {
     
+    public static final Logger COMMAND_LOGGER = CommonLoggers.commands();
```

Comment:
should this not be in CommonLoggers?

Reply:
The idea was to provide an alternative to accessing the field without going through `CommonLoggers`, ensuring that people could use the field reference and not worry about what the actual subsystem name is. This would allow us to potentially change it in the future to different or more specific subsystems if the need arises.

Would you prefer having `CommonLoggers.commands()` regardless?